envsubst/manifest/manifests.yaml.template

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
Did you perhaps copy from c2d? I don't think we should do that--this is already available in C2D and it also serves as examples.

We're also killing the ubbagent example here. I think the agreement was that we would just rename this and migrate it away from Makefiles, instead of changing the entire example.

The other thing we wanted to do was to make the examples really small to demonstrate the integration. We can do that once we figure out where to put the ubbagent example.